tests/functional/object/rgw/test_rgw_read_affinity.py

```diff
@@ -18,9 +17,9 @@
 log = logging.getLogger(__name__)
 
 CEPH_OBJECT_STORE = OCP(
-    kind="CephObjectStore", namespace=config.ENV_DATA["cluster_namespace"]
+    kind="CephObjectStore", namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
```

Comment:
as far as I know, from few version back when we started with custom namespace for ODF, we have changed most of `constants.OPENSHIFT_STORAGE_NAMESPACE` and `openshift-storage` namespace to `config.ENV_DATA["cluster_namespace"]`

so is this a valid fix. Please run this on other platform for verification